cmd/ddev/cmd/config.go

```diff
@@ -93,8 +93,8 @@ var ConfigCommand = &cobra.Command{
 				util.Failed("apptype must be drupal7, drupal8, or wordpress")
 			}
 
-			foundAppType, err := ddevapp.DetermineAppType(c.Docroot)
-			fullPath, _ := filepath.Abs(c.Docroot)
+			foundAppType, err := ddevapp.DetermineAppType(app.Docroot)
+			fullPath, _ := filepath.Abs(app.Docroot)
```

Comment:
I know this isn't a change, but it's odd we don't check capture/check err here.

Reply:
Added the check.